### ISSUE TITLE
Rename 'master' branch to 'main'

### ISSUE THREAD

**Issue by mattt** [SwiftDocOrg/swift-doc#188]
As discussed in https://github.com/SwiftDocOrg/swift-doc/pull/184#issuecomment-698620244.

**Comment by compnerd** [comment]
Just a note that this change will require a code change as well here: https://github.com/SwiftDocOrg/swift-doc/blob/307e486cd5615c05dfd9cf1266d6d207e29bdfb2/Sources/swift-doc/Subcommands/Generate.swift#L163

**Comment by mattt** [comment]
@compnerd Yep. Thanks for the reminder. We'll keep a `master` branch around for a while to make sure we don't break existing clients.  

Prospectively, that code is a holdover from a migration to package resources that we had to back out of. Obviously it'd be better if the CSS was bundled with the tool; I'll give that another try, maybe for 1.0.0.

**Comment by Lukas-Stuehrk** [comment]
Also related to #231. Bundling the CSS with the tool will also fix this problem.
